libsrc/meshing/basegeom.hpp

```diff
@@ -339,15 +339,12 @@ namespace netgen
   public:
     virtual ~GeometryRegisterArray()
     {
-      for (int i = 0; i < Size(); i++)
-        delete (*this)[i];
+      DeleteAll();
     }
-
     virtual shared_ptr<NetgenGeometry> LoadFromMeshFile (istream & ist) const;
```

Comment:
LoadFromMeshFile is not reimplemented, but delegated to  the registered GeometryRegister::LoadFromMeshFile.
```suggestion
    shared_ptr<NetgenGeometry> LoadFromMeshFile (istream & ist) const;
```